cdk/lib/cdk-playground.ts

```diff
@@ -24,6 +24,9 @@ export class CdkPlayground extends GuStack {
 
 		const ec2App = 'cdk-playground';
 		const ec2AppDomainName = 'cdk-playground.gutools.co.uk';
+    //New domain added for Fastly disaster recovery testing. Will be deleted after this testing is complete
+    //todo - delete fastlyTestDomainName once the Fastly disaster recovery testing is complete.
+    const fastlyDRTestDomainName = 'cdk-status-resources.guardianapis.com';
```

Comment:
Updating to use guardianapis.com as this also allows any prefix and also is a better match for the content served when you hit this endpoint.